src/pcapi/core/offers/api.py

```diff
@@ -282,10 +333,92 @@ def as_utc_without_timezone(d: datetime.datetime) -> datetime.datetime:
 
     if feature_queries.is_active(FeatureToggle.SYNCHRONIZE_ALGOLIA):
         redis.add_offer_id(client=app.redis_client, offer_id=stock.offerId)
-
     return stock
 
 
+def bulk_create_edit_stocks(stockDataList: List[Union[StockCreationBodyModel, StockEditionBodyModel]]):
+    stocks = []
+    edited_stocks = []
+    edited_stocks_previous_beginnings = {}
+    offerId = None
+    stock_errors = [{}] * len(stockDataList)
+    has_errors = False
+    db.session.begin_nested()
+
+    for idx, data in enumerate(stockDataList):
+        try:
+            if isinstance(data, StockEditionBodyModel):
+                stock = Stock.queryNotSoftDeleted().filter_by(id=data.id).join(Offer, VenueSQLEntity).first_or_404()
+                edited_stocks_previous_beginnings[stock.id] = stock.beginningDatetime
+                if offerId == None:
```

Comment:
```suggestion
                if offerId is None:
```

Les deux sont corrects, mais `is None` est plus idiomatique.